Tests/Shared.Specs/SimpleTimeSpanAssertionSpecs.cs

```diff
@@ -40,6 +40,25 @@ public void When_asserting_negative_value_to_be_positive_it_should_fail()
             act.Should().Throw<XunitException>();
         }
 
+        [Fact]
+        public void When_asserting_null_value_to_be_positive_it_should_fail()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            TimeSpan? nullTimeSpan = null;
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Act
+            //-----------------------------------------------------------------------------------------------------------
+            Action act = () => nullTimeSpan.Should().BePositive();
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Assert
+            //-----------------------------------------------------------------------------------------------------------
+            act.Should().Throw<XunitException>();
```

Comment:
Usually we do verify that at least parts of the exception message make sense

Reply:
Thanks
I've updated the assertions with separated `null` check statements and added assertions on the failure message in the tests.